ingestor/.chalice/policy-yankee.json

```diff
@@ -24,15 +24,20 @@
       "Resource": ["arn:aws:dynamodb:us-east-1:473352343756:table/ShuttleTravelTimes"]
     },
     {
-      "Action": [
-        "s3:GetObject",
```

Comment:
Ideally I'd prefer we specified instead of wildcard

Reply:
yeah sure! I can be more explicit. I just couldn't remember which permission `ListObjects` fell under honestly